### ISSUE TITLE
HTML Formatter removes content of script tag

### ISSUE THREAD

**Issue by ezra100** [Microsoft/vscode#39706]
- VSCode Version: 1.18.1
- OS Version: Windows 10

Steps to Reproduce:

1. Open a new file and change the language to HTML.
2. Insert the following code (**Make sure that you leave a space between the script opening and closing tags**):
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"> </script>
    </head>
</html>
```
3. Format code (Ctrl+Alt+F)

What should happen:
The code should be formatted without changing the content.

What actually happens:
The code is formatted but the opening script tag and its content is removed. Final results is:
```
<!DOCTYPE html>
<html lang="en">

<head>
    </script>
</head>

</html>
```

Reproduces without extensions:  The formatter seems to be a part of an extension that comes with VSCode, so with the extensions disabled Ctrl+Alt+F does nothing.

**Comment by maxibanki** [comment]
Please provide us an list of your installed extension. This can be easily done by using the `Report Issues` button in VS Code: Help => Report Issues menu.

**Comment by ezra100** [comment]
These are my extensions:

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.1.0
vscode-css-formatter|aes|1.0.1
vscode-eslint|dba|1.4.3
xml|Dot|1.9.2
vscode-html-css|ecm|0.1.8
tslint|eg2|1.0.24
python|ms-|0.8.0
debugger-for-chrome|msj|3.5.0
ejs-language-support|Qas|0.0.1

**Comment by ezra100** [comment]
I've disabled all HTML/css related extensions, but the bug is still there.

**Comment by aeschli** [comment]
duplicate of #36574, fixed in 1.19
